packages/components/src/text-input/docs/TextInput.stories.mdx

```diff
@@ -26,7 +26,7 @@ A default text input.
 
 <Preview>
     <Story name="default">
-        <TextInput placeholder="Where to?" />
+        <TextInput autoFocus placeholder="Where to?" />
```

Comment:
to remove

Reply:
Why? It's a standard feature of all our components.

Reply:
The boolean form of `autoFocus` is easy to integrate to the hook we discussed earlier today, the delayed form will be harder